up-and-running/customization.md

```diff
@@ -1,31 +1,145 @@
 # Customizating Server Configuration
 
 You can customize the server's configuration by visiting npm
-Enterprise's admin console (_http://myreg.mycompany.com:8800_).
+Enterprise's admin console on `:8800`
 
-## Authentication
+  ![Admin Console](/gitbook/images/02-settings.png)
 
-Authentication method.
+## Your First Time Running npm Enterprise
 
-Possible values: [`"GitHub"`](/enterprise/github), [`"Open"`](/enterprise/no-authentication), "LDAP"
+The first time you visit your appliance on port `:8800`, there is some
+initial configuration that you will need to perform:
 
-## Upstream registry
+1. because no certificate has been configured for your appliance, you
+  will need to proceed past a certificate warning in the browser:
 
-Configure npm Enterprise to replicate from an upstream registry.
-This can be useful for creating backups.
+  ![Admin Console](/gitbook/images/01-security-warning.png)
 
-## Storage
+2. next you will be prompted to upload a certificate, or to generate
+  a self-signed certificate (this certificate will be used going forward
+  when accessing the admin console).
```

Comment:
Should we perhaps mention that the cert will **not** be used for the npme registry or website?